app/models.py

```diff
@@ -57,12 +58,56 @@ def followed_posts(self):
         own = Post.query.filter_by(user_id=self.id)
         return followed.union(own).order_by(Post.timestamp.desc())
 
-class Post(db.Model):
+
+class SearchableMixin(object):
```

Comment:
'object' can be omitted
https://www.quora.com/Why-do-classes-inherit-an-object-base-class-in-Python